desktop/assets/analytics.coffee

```diff
@@ -3,10 +3,16 @@ mediator = require '../lib/mediator.coffee'
 setupSplitTests = require '../components/split_test/setup.coffee'
 window._ = require 'underscore'
 window.Cookies = require 'cookies-js'
+Events = require('@artsy/reaction-force/dist/Utils/Events.js').default
 
+# All Force mediator events can be hooked into for tracking purposes
 mediator.on 'all', (name, data) ->
   analyticsHooks.trigger "mediator:#{name}", data
 
+# All Reaction events are sent directly to Segment
+Events.onEvent (data) =>
+  analytics.track data.action, _.omit data, 'action'
+
```

Comment:
Love the simplicity of this!

Reply:
Why do we omit `action` from data here @craigspaeth ?

Reply:
Just because we're sending it as the event name... React Tracking's conventions is:

```
{
  action: "event name",
  extra: "data"
}
```

and Segment's convention is:

```
"event name", { extra: "data" }
```

When the data gets to Redshift/Looker the event name is also added a column, so no need to include a duplicate `action` field as well.